src/pantry.html

```diff
@@ -18,7 +18,7 @@ <h1 class="open-menu-title x">Menu</h1>
     </header>
     <section class="open-menu-selections x">
       <article class="menu-my-pantry menu-selection x">
-        <h3 class="menu-my-pantry-title">My Pantry</h2>
+        <h3 class="menu-all-recipe-title">All Recipes</h2>
```

Comment:
is this the 'back to home' version of what we were talking about earlier? does this take the user back to the main page?